src/test/java/au/org/emii/MyIT.java

```diff
@@ -0,0 +1,111 @@
+
+package au.org.emii.ncdfgenerator;
```

Comment:
Package name doesn't match directory structure causing compile errors for me.  You may have fixed this...